docs/app/components/api-property/api-property.component.less

```diff
@@ -0,0 +1,17 @@
+@import '../../assets/styles/variables.less';
+
+.api-property-name,
+.api-property-details {
+    border: 1px solid @table-grid-border;
+    padding: 8px;
+    vertical-align: initial;
+
+    .api-property-metadata-value {
+        color: #c7254e;
+        font-style: normal;
```

Comment:
I just noticed this. If you arrange the spans side-by-side, you won't have to override the italic style here.